src/datadog/tracer_config.cpp

```diff
@@ -21,48 +21,86 @@ namespace datadog {
 namespace tracing {
 namespace {
 
-bool falsy(StringView text) {
-  auto lower = std::string{text};
-  to_lower(lower);
-  return lower == "0" || lower == "false" || lower == "no";
+std::string to_string(bool b) { return b ? "true" : "false"; }
+
+// TODO: use `to_chars`
```

Comment:
To avoid:
<img width="835" alt="Screenshot 2024-03-01 at 11 16 16" src="https://github.com/DataDog/dd-trace-cpp/assets/204994/e5c8d52d-ddcb-4af6-8535-ac7ed42ea634">